lib/mongorito.js

```diff
@@ -56,21 +71,23 @@ class Mongorito {
         options = arg;
       }
     });
-
-    // convert mongo:// urls to monk-supported ones
     urls = urls.map(function (url) {
-      return url.replace(/^mongo\:\/\//, '');
+      if(!url.startsWith('mongodb://')){
+        url = 'mongodb://' + url;
+      }
+      return url;
     });
 
-    let db = monk(urls, options);
-
-    // if there is already a connection
-    // don't overwrite it with a new one
-    if (!this.db) {
-      this.db = db;
-    }
-
-    return db;
+    return new Promise(function(resolve, reject){
```

Comment:
I know it's a promise but I can't just return it because of the need to set the instance.db. Whether it's set on the this instance or the signleton instance I'm using now this problem still exists.